packages/suite/src/components/suite/CoinjoinStatusBar.tsx

```diff
@@ -0,0 +1,137 @@
+import { Account, CoinjoinSession, WalletParams } from '@suite-common/wallet-types';
+import { CountdownTimer } from './Timer';
+import { useActions } from '@suite-hooks/useActions';
+import * as routerActions from '@suite-actions/routerActions';
+import { useSelector } from '@suite-hooks/useSelector';
+
+const SPACING = 6;
+
+const Container = styled.div`
+    display: flex;
+    align-items: center;
+    height: 28px;
+    padding: 0 ${SPACING}px;
+    background: ${({ theme }) => theme.BG_WHITE};
+    border-bottom: 1px solid ${({ theme }) => theme.STROKE_LIGHT_GREY};
+    font-size: ${variables.FONT_SIZE.TINY};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+`;
+
+const ProgressPie = styled.div<{ progress: number }>`
+    width: 16px;
+    height: 16px;
+    margin-right: ${SPACING}px;
+    border-radius: 50%;
+    background: ${({ theme, progress }) =>
+        `conic-gradient(${theme.BG_GREEN} ${3.6 * progress}deg, ${theme.STROKE_GREY} 0)`};
+`;
+
+const StatusText = styled.span`
+    color: ${({ theme }) => theme.TYPE_GREEN};
+`;
+
+const NextRoundNote = styled.span`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+const Separator = styled.span`
+    margin: 0 ${SPACING / 2}px;
+`;
+
+const ViewButton = styled(Button)`
+    height: 20px;
+    margin-left: auto;
+`;
+
+enum MockCoinjoinPhase {
+    Starting,
+    Ongoing,
+    Error,
+}
+
+const STATUS_MESSAGES = {
+    [MockCoinjoinPhase.Starting]: 'Collecting inputs',
+    [MockCoinjoinPhase.Ongoing]: 'Joining coins',
+    [MockCoinjoinPhase.Error]: 'U poor now',
+};
+
+interface CoinjoinStatusBarProps {
+    accountKey: string;
+    session?: CoinjoinSession;
+}
+
+export const CoinjoinStatusBar = ({ accountKey, session }: CoinjoinStatusBarProps) => {
+    const accounts = useSelector(state => state.wallet.accounts);
+    const routerParams = useSelector(state => state.router.params) as WalletParams;
+
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    const phase = MockCoinjoinPhase.Ongoing;
+    const progress = 36; //  current_phase / (total_phases / 100)
+
+    const { symbol, index, accountType } = accounts.find(
+        ({ key }) => key === accountKey,
+    ) as Account;
+
+    const {
+        symbol: symbolParam,
+        accountIndex: indexParam,
+        accountType: accountTypeParam,
+    } = routerParams || {};
+
+    const isOnAccounPage =
+        symbolParam === symbol && indexParam === index && accountTypeParam === accountType;
+
+    return (
+        <Container>
+            <ProgressPie progress={progress} />
+
+            <StatusText>
+                {STATUS_MESSAGES[(session?.phase as MockCoinjoinPhase) || phase]}
+
+                <Separator>•</Separator>
+
+                <Translation
+                    id="TR_COINJOIN_COUNTDOWN"
+                    values={{ rounds: '8' }} // TODO: How to get?
+                />
+            </StatusText>
+
+            {session?.deadline && (
```

Comment:
How should the timer behave when a round is delayed? Probably display a different message when a deadline is missed @hynek-jina.

Reply:
Can we just change the timer whenever we get better estimate? Why should be the round delayed?

Reply:
Szymon said that he can calculate any deadline, currently it's till the end of the **phase**, calculated at the beginning of each **phase**. If we want to decrease probability of running out of time we can recalculate the deadline more often. Also, on the designs it's time till the end of the **round** not **phase**. We should agree on one thing and either me or Szymon will change the code.

Reply:
I think we should check the timer regularly. CoinJoin round can start earlier if there is enough inputs ready. Can we check that for example every 5s?
TIme estime for all CoinJoin rounds can be just on the begining of each round..

Reply:
For now, I have the "x rounds left" clause instead of "x time left" since I would need the final specifications for the total estimate. And please let Szymon know regarding the phase deadline recalculation. I'd say 5 sec is too extreme. 
The situation I wanted to avoid initially was a phase taking longer than a deadline predicted. It's better to show "1min left" for 3 mins than to come up with additional logic for displaying something like "oh, it's taking a bit longer than it should".

Reply:
I think that lastly the outcome of our discussion was to estimate even total time (like `number of estimated rounds` * `some time constant` (ie 2 hours and 30 minutes for the beginning → before we collect some data on that)

@szymonlesisz Please how often would you check the remaining time for the next CoinJoin round?

Reply:
Sure, if we agreed on 2.5h I can add that.

Reply:
Yes, It's just a placeholder for now. And I hope we can be more precise soon.